src/App.tsx

```diff
@@ -9,38 +9,32 @@ import ResetPass from "./modules/Auth/Components/ResetPass/ResetPass";
 import Registration from "./modules/Auth/Components/Registration/Registration";
 import VerifyAccount from "./modules/Auth/Components/VerifyAccount/VerifyAccount";
 import ChangePassword from "./modules/Auth/Components/ChangePassword/ChangePassword";
-import MasterLayout from "./modules/Shared/Components/MasterLayout/MasterLayout";
-import DashBoard from "./modules/DashBoard/Components/DashBoard";
-import Rooms from "./modules/Rooms/Rooms";
-import Booking from "./modules/Booking/Booking";
-import Ads from "./modules/Ads/Ads";
-import User from "./modules/User/User";
-import Favorite_rooms from "./modules/Favorite_rooms/Favorite_rooms";
-import Room_comments from "./modules/Room_comments/Room_comments";
-import Room_reviews from "./modules/Room_reviews/Room_reviews";
-import Room_facility from "./modules/Room_facility/Room_facility";
-import { useContext } from "react";
+import DashBoard from "./modules/Admin/Components/DashBoard/Components/DashBoard";
+import AdsList from "./modules/Admin/Components/Ads/AdsList/AdsList";
+import AdsForm from "./modules/Admin/Components/Ads/AdsForm/AdsForm";
+import BookingList from "./modules/Admin/Components/BookingList/BookingList";
+import RoomFacilitiesList from "./modules/Admin/Components/RoomFacilities/RoomFacilitiesList/RoomFacilitiesList";
+import RoomsList from "./modules/Admin/Components/Rooms/RoomsList/RoomsList";
+import UserList from "./modules/Admin/Components/UserList/UserList";
+import MasterAdminLayout from "./modules/Shared/Components/MasterAdminLayout/MasterAdminLayout";
+import MasterUserLayout from "./modules/Shared/Components/MasterUserLayout/MasterUserLayout";
+import HomePage from "./modules/User/Components/HomePage/HomePage";
+import RoomDetailsPage from "./modules/User/Components/RoomDetailsPage/RoomDetailsPage";
+import BookingPage from "./modules/User/Components/BookingPage/BookingPage";
+import ExploarePage from "./modules/User/Components/ExploarePage/ExploarePage";
+import FavoriteRoomPage from "./modules/User/Components/FavoriteRoomPage/FavoriteRoomPage";
 import ProtectedRoute from "./modules/Auth/Components/ProtectedRoute/ProtectedRoute";
+import { useContext } from "react";
 import { AuthContext } from "./context/AuthContext";
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 function App() {
-
-  const LoginData = useContext(AuthContext);
-
-
-  // const [loginData, setLoginData] = useState(null);
-  // const saveLoginData = () => {
-  //   const deCodedToken = localStorage.getItem("token");
-  //   if (deCodedToken) {
-  //     const enCodedToken = jwtDecode(deCodedToken);
-  //     setLoginData(enCodedToken);
-  //     console.log(enCodedToken);
-  //   }
-  // };
+  const loginData = useContext(AuthContext); // Get loginData from context
 
   const router = createBrowserRouter([
     {
-      path: "",
+      path: "login",
```

Comment:
This will mean that to reach for registeration, the url will be 'login/registeration' which is not good. To make the landing page appear first, we just need to make it the first router with path "".

Reply:
done